atest/robot/output/format_log_to_console.robot

```diff
@@ -0,0 +1,15 @@
+*** Settings ***
+Library    Process
+Library    OperatingSystem
+*** Test Cases *** 
+Center align log to console with star padding
+    Log to console  hello console  format=*^78
+Right align log to console with hastag padding
+    Log to console  test123$#@!$  format=#>32
+Right align log to console with space padding
+    Log To Console  +000000120\./asdf  format=>30
+Test if formatting works with no_newline argument
+    Log To Console  message starts here,  format=>44  no_newline=true
+    Log To Console  this sentence should be on the same sentence as "message starts here"
+Test if formatting works with stderr argument
+    Log to console  log to stderr  format=>44  stream=STDERR
```

Comment:
These tests belong here:
https://github.com/robotframework/robotframework/blob/master/atest/testdata/standard_libraries/builtin/log.robot

Then you also need to add matching test to the `atest/robot` side to validating the logged output. See [atest/README.rst](https://github.com/robotframework/robotframework/blob/master/atest/README.rst) for more information about our acceptance system. If you need help with it, don't hesitate to ask here or on Slack.